Formula/earthly.rb

```diff
@@ -35,13 +37,14 @@ def install
   end
 
   test do
-    (testpath/"build.earthly").write <<~EOS
-
-      default:
+    # earthly requires docker to run; therefore doing a complete end-to-end test here is not
+    # possible; however the "earthly ls" command is able to run without docker.
+    (testpath/"Earthfile").write <<~EOS
+      VERSION 0.6
+      mytesttarget:
       \tRUN echo Homebrew
     EOS
-
-    output = shell_output("#{bin}/earthly --buildkit-host 127.0.0.1 +default 2>&1", 6).strip
-    assert_match "buildkitd failed to start", output
+    output = shell_output("#{bin}/earthly ls").strip
+    assert_match "+mytesttarget", output
```

Comment:
Since match is a regex or partial match this isn't needed.
```suggestion
    output = shell_output("#{bin}/earthly ls")
    assert_match "+mytesttarget", output
```

Reply:
good point!